panel/src/components/Layout/ListItem.vue

```diff
@@ -17,12 +17,18 @@
       </span>
     </k-link>
     <nav class="k-list-item-options">
+      <k-button
+        v-if="flag"
+        v-bind="flag"
+        class="k-list-item-status"
+        @click="flag.click"
+      />
+      <k-status-icon
+        v-else-if="statusIcon"
+        v-bind="statusIcon"
+        class="k-list-item-status"
+      />
       <slot name="options">
-        <k-status-icon
-          v-if="flag"
-          v-bind="flag"
-          class="k-list-item-status"
-        />
```

Comment:
Why was the component moved out of the slot?

Reply:
Yikes, good catch!